.prettierrc.json

```diff
@@ -1,5 +1 @@
-{
-  "trailingComma": "none",
-  "singleQuote": true,
-  "semi": false
```

Comment:
This was being ignored anyway, so I removed it. This also matches what `ucp-hosted-apps` has, which I think is a cleaner setup. We should align this project with that one in the future.